app/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools">
+    xmlns:tools="http://schemas.android.com/tools"
+    package="com.example.evently">
```

Comment:
You aren't supposed to set this in modern gradle systems: https://developer.android.com/guide/topics/manifest/manifest-element#package

> In the Gradle-based build system, starting with AGP 7.3, don't set the package value in the source manifest file directly.